packages/core/test/foundry/optimistic-asserter/OptimisticAsserter.Maintenance.t.sol

```diff
@@ -100,5 +100,6 @@ contract MaintenanceTest is Common {
         );
         vm.stopPrank();
         assertTrue(optimisticAsserter.cachedIdentifiers(newIdentifier));
+        assertTrue(false);
```

Comment:
nit: what is this change?

Reply:
this was testing that things produced the correct error. the PR is still in draft ;)

Reply:
No, no worries, just didn't know if it was intentional or not! Makes sense. Disregard :)